controller/rest/rest.go

```diff
@@ -1716,7 +1716,8 @@ func StartRESTServer(isNewCluster bool, isLead bool) {
 	r.GET("/v1/custom_check/:group", handlerCustomCheckShow)
 	r.GET("/v1/custom_check", handlerCustomCheckList)
 	r.PATCH("/v1/custom_check/:group", handlerCustomCheckConfig)
-	r.GET("/v1/compliance/profile", handlerComplianceProfileList) // Only default is accepted, so not POST/DELETE
+	r.GET("/v1/compliance/available_filter", handlerGetAvaiableComplianceFilter) // Skip API document, use internally
```

Comment:
This new API is missed in controller/access/access.go & access_test.go
(search for "v1/compliance/profile" in both files for reference)